### PR TITLE
replace sub-heading with sub-title

### PR DESCRIPTION
**Pull request by raynamharris**
as per issue #224

### CODE REVIEW

File: _episodes/04-formatting.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ authors should use only:
 
 Authors should *not* use:
 
-*   sub-headings
+*   sub-titles (use H2 subheadings instead)
```
Comment (katrinleinweber) [inline _episodes/04-formatting.md]: How about being explicit about `##` in the .(R)md files here?

Comment (maxim-belkin) [reply _episodes/04-formatting.md]: Good suggestion!

Comment (raynamharris) [reply _episodes/04-formatting.md]: how is this @katrinleinweber https://github.com/carpentries/lesson-example/pull/225/commits/6463d780e89a7fc8127422cc6ed137a45c738dd9

Comment (brownsarahm) [reply _episodes/04-formatting.md]: should the h2 headings be added to  the list of things to be used above?

Comment (maxim-belkin) [reply _episodes/04-formatting.md]: > should the h2 headings be added to the list of things to be used above?

Makes sense, I think.

**Comment by raynamharris** [comment]
with approval of @katrinleinweber, I think this is okay to merge

**Pull request merged**